cmd/checkmetrics/README.md

```diff
@@ -0,0 +1,124 @@
+# checkmetrics
+
+## Overview
+
+The `checkmetrics` tool is used to check the metrics CSV results
+files against a TOML file that contains baseline expectations for
+the results.
+
+`checkmetrics` checks for a matching CSV file for each entry in the
+TOML file. Failure to find a matching CSV is classified as a failure.
+
+`checkmetrics` will continue to process all entries in the TOML file
+and report its final results in a summary table.
+
+Each CSV metrics file must container at least one header line, with the
```

Comment:
Fixed. Found another similar one in the help as well, and fixed here and in the code